src/nvim/if_cscope.c

```diff
@@ -839,7 +839,7 @@ static int cs_create_connection(size_t i)
     si.hStdOutput = stdout_wr;
     si.hStdError  = stdout_wr;
     si.hStdInput  = stdin_rd;
-    created = CreateProcess(NULL, cmd, NULL, NULL, TRUE, CREATE_NEW_CONSOLE,
+    created = CreateProcess(NULL, cmd, NULL, NULL, true, CREATE_NEW_CONSOLE,
```

Comment:
If code still compiles if `TRUE` is used here and the defines are removed from `vim.h`, then I guess there is no guarantee that passing `true` is the same as passing `TRUE` here, as it is possible that system headers define `TRUE` and `true` differently.

Reply:
That's always been the case since the defines were always guarded:

```c
#ifndef TRUE
# define FALSE  0           // note: this is an int, not a long!
# define TRUE   1
#endif
```

So I think we can assume TRUE is always defined as 1.

Reply:
I search with sourcegraph and I can indeed find many examples where `true` is passed to `CreateProess()`, so I guess this is fine.